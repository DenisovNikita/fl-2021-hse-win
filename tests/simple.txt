0 to 1 by 5 0____1
1
2

