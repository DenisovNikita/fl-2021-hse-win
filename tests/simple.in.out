DFA {
  Alphabet 2: {a, b}
  States 2: {0, 1}
  Transpositions:
    between states 0 and 0: {}
    between states 0 and 1: {a, b}
    between states 1 and 0: {a, a}
    between states 1 and 1: {}
  Terminals 1: {0}
}
